apps/production/views.py

```
@@ -1549,6 +1579,108 @@ def _store_added_ingredient(order_obj, ingredient_obj, quantity_new):
                 if _store_added_ingredient(order_obj, ingredient_obj, quantity_new):
                     added_count += 1
 
+            # Zpracování nových položek pro polévku (jídlo „Polévka" – v
```

Ruff (0.15.21)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)